catalog/pages/inputs/index.md

```diff
@@ -291,3 +295,77 @@ span: 6
     </Row>
 </Container>
 ```
+
+## Drop Down Group
+
+### Props
+
+```table
+span: 6
+rows:
+  - Prop: onChange
+    Type: string
+    Default: top
+    Notes: Can be top or left
+  - Prop: variant
+    Type: string
+    Default: 0
+    Notes: one of 0 or 1
```

Comment:
Can you please provide additional documentation regarding the purposes of variants 0 and 1? Also, would it be more informative for the variants to have sematic rather than numeric names?